src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -709,7 +736,7 @@ internal void DownloadAndInstall(PackageDownloadHandle packageDownloadHandle, st
                     try
                     {
                         packageDownloadHandle.Done(pathDl);
-                        var firstOrDefault = PackageManagerExtension.PackageLoader.LocalPackages.FirstOrDefault(pkg => pkg.ID == packageDownloadHandle.Id);
+                        var firstOrDefault = PackageManagerExtension.PackageLoader.LocalPackages.FirstOrDefault(pkg => pkg.Name == packageDownloadHandle.Name);
```

Comment:
same issue with ID vs Name
broken here https://github.com/DynamoDS/Dynamo/pull/10544/files#diff-10009027dea2b20f51c89c7b0306d25ac8d92dbfc0fbfb891fdfa171eba8df1fR707

Reply:
Can you give an example of what a package `id` looks like vs. a package `name`?

Reply:
the ID is the PM server's ID (a GUID)
The name is any package's name